src/router/metadata/cap.rest.resource.template.ts

```diff
@@ -75,16 +74,9 @@ export function makeGenericResources(
 
 export async function makeResource(resourceType: string, resource: Resource) {
     const resourceOperations: any[] = makeOperation(resource.operations);
-    const updateCreate: boolean = resource.operations.includes('update');
     const hasTypeSearch: boolean = resource.operations.includes('search-type');
 
     const capabilities: SearchCapabilityStatement = hasTypeSearch ? await resource.typeSearch.getCapabilities() : {};
 
-    return makeResourceObject(
-        resourceType,
-        resourceOperations,
-        updateCreate,
-        hasTypeSearch,
-        capabilities[resourceType],
-    );
+    return makeResourceObject(resourceType, resourceOperations, false, hasTypeSearch, capabilities[resourceType]);

```

Comment:
This is more accurate right now, but we should stop using hardcoded values.

We should be able to use the `updateCreateSupported` field from the `Persistence` interface to populate this https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/blob/mainline/src/dataServices/dynamoDbDataService.ts#L62

Reply:
updated